### ISSUE TITLE
[Error] Authentification is required 

### ISSUE THREAD

**Issue by sidki** [poiuytrez/AndroidInAppBilling#94]
Whenever I click purchase on any item it shows an error that says ***"Authentication is required. You need to sign into your Google account"*** and I'm already signed in.
Then when I alert the response I get ***"Error Purshasing: IabResult: User canceled. (response: -1005:User cancelled)"***

**Comment by jreynerio** [comment]
Hi sidki I have the same error, do you fixed ? 

Regards!

**Comment by sidki** [comment]
Nope, I used this plugin instead and worked well for me https://github.com/Fovea/cordova-plugin-purchase-demo
